vivisect/symboliks/common.py

```diff
@@ -81,6 +81,7 @@ def __init__(self):
         self.kids          = []
         self.parents       = []
         self.cache         = {}
+        self._reduced       = False
```

Comment:
Re your onus on the caller comment above, what if we made it an option within walkTree()?

walkTree(once=True)  # automatically track ones we've already walked and do not fire the callback more than once for them etc...